src/components/ErrorMessage/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./ErrorMessage";
```

Comment:
This is a question more than a comment. Why is your 'export' in a separate index.js file as well as in the original component?